src/messaging/messaging.ts

```diff
@@ -221,6 +223,22 @@ export class Messaging {
     return this.appInternal;
   }
 
+  /**
+   * Enables the use of the legacy HTTP/1.1 transport for sendEach() and sendEachForMulticast().
+   * 
+   * @example
+   * ```javascript
+   * const messaging = getMessaging(app);
+   * messaging.enableLegacyTransport();
+   * messaging.sendEach(messages);
+   * ```
+   * 
+   * @deprecated This is to be removed once the HTTP/2 transport is universally safe.
```

Comment:
`This is to be removed once the HTTP/2 transport implementation is universally safe.` ?

Reply:
Safe here was to imply when the HTTP/2 implementation was stable enough where it was on par  with the HTTP/1.1 implementation and no longer needed an emergency back up in case some functionality was completely covered.

Went with the following but open to any suggestions here: 
`This is to be removed once the HTTP/2 transport implementation reaches the same stability as the legacy HTTP/1.1 implementation.`

Reply:
That works for me!

If it's easily done, I'd tone down to "This will be removed when..."